test/workbox-window/window/test-Workbox.mjs

```diff
@@ -301,24 +301,27 @@ describe(`[workbox-window] Workbox`, function() {
       expect(reg.update.callCount).to.equal(1);
     });
 
-    it(`triggers an updatefound event if the SW was udpated`, async function() {
+    it(`triggers an updatefound event if the SW was updated`, async function() {
       const scriptURL = navigator.serviceWorker.controller.scriptURL;
 
       const wb = new Workbox(scriptURL);
+
       const reg = await wb.register();
+      const updatefoundPromise = new Promise((resolve) => {
```

Comment:
This test from yesterday needed a bit more cleaning up, as there was a race condition in which the `updatefound` listener might not have been registered in time to catch the update. Registering the listener early and then waiting later on for the promise resolution should fix that.